packages/block-library/src/image/edit.native.js

```diff
@@ -430,6 +434,48 @@ export class ImageEdit extends Component {
 		this.onSetSizeSlug( newValue );
 	}
 
+	onSetFeatured( mediaId ) {
+		const { closeSettingsBottomSheet } = this.props;
+		setFeaturedImage( mediaId );
+		closeSettingsBottomSheet();
+	}
+
+	getSetFeaturedButton( isFeaturedImage ) {
+		const { attributes, getStylesFromColorScheme } = this.props;
+
+		const setFeaturedButtonStyle = getStylesFromColorScheme(
+			styles.setFeaturedButton,
+			styles.setFeaturedButtonDark
+		);
+
+		const setFeaturedButton = () => (
+			<BottomSheet.Cell
+				label={ __( 'Remove as Featured Image ' ) }
+				labelStyle={ [
+					setFeaturedButtonStyle,
+					styles.removeFeaturedButton,
+				] }
+				onPress={ () => this.onSetFeatured( 0 ) }
```

Comment:
@SiobhyB similar to the comment made on the WP Android PR https://github.com/wordpress-mobile/WordPress-Android/pull/14503#discussion_r638422315 what do you think about hardcoding the default media id as a constant for clarity?

Reply:
@jd-alexander, that makes sense! I've gone ahead to add the constant in https://github.com/WordPress/gutenberg/pull/31705/commits/5681237eca310bc770ad7bc36ef374d260a8b625, does that seem right to you?

Reply:
Just to update: I went ahead to move the constant from the function to the top of the file in https://github.com/WordPress/gutenberg/pull/31705/commits/7f6ad6b758d43e13e2879d9933cc42221a960ddb, as this seemed to match more closely with the pattern I see for other constants. Let me know if there's a better way to declare constants in Gutenberg or if there's something that can be done better here, though!

Reply:
Looks like all feedback here has already been addressed and we now have a constant to represent the `0` value. Will mark this convo as resolved :)